docs/changes/stats/12146.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed a bug in ``biweight_midvariance`` and ``biweight_scale`` where
+output data units would be dropped for constant data.
```

Comment:
Maybe also mention the `np.nan` case here?

Reply:
done